packages/shared/src/modules/contributors/libs/validation-schemas/contributor-split.validation-schema.ts

```diff
@@ -0,0 +1,29 @@
+import { z } from "zod";
+
+import {
+	ContributorValidationMessage,
+	ContributorValidationRule,
+} from "../enums/enums.js";
+import { type ContributorSplitRequestDto } from "../types/types.js";
+
+const contributorSplit: z.ZodType<ContributorSplitRequestDto> = z
+	.object({
+		emailId: z
```

Comment:
let's rename it to `gitEmailId` instead to avoid confusion, and the same for error message 
```suggestion
		gitEmailId: z
```